pyro/infer/elbo.py

```diff
@@ -5,13 +5,26 @@
 import warnings
 from abc import ABCMeta, abstractmethod
 
+import torch
+
 import pyro
 import pyro.poutine as poutine
 from pyro.infer.util import is_validation_enabled
 from pyro.poutine.util import prune_subsample_sites
 from pyro.util import check_site_shape
 
 
+class _ELBOModule(torch.nn.Module):
```

Comment:
nit: consider going ahead and name this publicly `ELBOModule`. I expect users may want to `torch.save` and inspect these things. And looking back I wish we had just made the `torch.constraints` classes public, since people now treat them as public even though there are underescores.

Reply:
Good point, done